Mathlib/Data/Nat/Choose/Central.lean

```diff
@@ -18,36 +19,40 @@ This file proves properties of the central binomial coefficients (that is, `nat.
 
 ## Main definition and results
 
-* `nat.central_binom`: the central binomial coefficient, `(2 * n).choose n`.
-* `nat.succ_mul_central_binom_succ`: the inductive relationship between successive central binomial
+* `Nat.centralBinom`: the central binomial coefficient, `(2 * n).choose n`.
+* `Nat.succ_mul_centralBinom_succ`: the inductive relationship between successive central binomial
   coefficients.
-* `nat.four_pow_lt_mul_central_binom`: an exponential lower bound on the central binomial
+* `Nat.four_pow_lt_mul_centralBinom`: an exponential lower bound on the central binomial
   coefficient.
-* `succ_dvd_central_binom`: The result that `n+1 ∣ n.central_binom`, ensuring that the explicit
+* `succ_dvd_centralBinom`: The result that `n+1 ∣ n.centralBinom`, ensuring that the explicit
   definition of the Catalan numbers is integer-valued.
 -/
 
 
 namespace Nat
 
-/-- The central binomial coefficient, `nat.choose (2 * n) n`.
+/-- The central binomial coefficient, `Nat.choose (2 * n) n`.
 -/
 def centralBinom (n : ℕ) :=
   (2 * n).choose n
 #align nat.central_binom Nat.centralBinom
 
+/-- expand centralBinom to its definition. -/
```

Comment:
```suggestion
/-- expand `centralBinom` to its definition. -/
```
Also elsewhere. I applaud the fact that you want to improve documentation. But I would rather encourage you to *not* fix documentation while porting. Because it means an extra review duty while reviewing porting PRs. Let's stick to doing one thing at a time.